### ISSUE TITLE
Set per-site Drush cache config

### ISSUE THREAD

**Issue by richardbporter** [uiowa/uiowa#4839]
Blocked by #4789 

Create a new PR using #3383 as a guide.

The PR should:
- [x] Remove env variables being set in BLT tasks [591f8f9](https://github.com/uiowa/uiowa/pull/3383/commits/591f8f9b6bb717731ed287d56e9484269f7c6915)
- ~~Create a BLT update hook to set site-specific config for all multisites [91788d9](https://github.com/uiowa/uiowa/pull/3383/commits/91788d9bad7a66d7ce0fb4f370a5aeea6b18eadb)~~
- ~~Ensure the same file is created for new multisites in `blt umc` command.~~
- ~~Add a filesystem test for said file.~~
- [x] Remove unused drush config [51f32f7](https://github.com/uiowa/uiowa/pull/3383/commits/51f32f753f53060b3c71c9f3426dbec9dd27b9b7)
- [x] Investigate BLT environment detector usage and determine if there is a way to set AH vars in DDev without breaking stuff. That is, it'd be nice to expand the environment variables used in the drush config files locally. [e16569d](https://github.com/uiowa/uiowa/pull/3383/commits/e16569da74711e1a65c52cb5e689e130436ec7d6)
- [x] Deploy to develop and test code deployments while running Drush commands like `blt ume cr`

~~It would good to do this while also pausing on any new site provisions until it is merged.~~

**Comment by richardbporter** [comment]
>Investigate BLT environment detector usage and determine if there is a way to set AH vars in DDev without breaking stuff. That is, it'd be nice to expand the environment variables used in the drush config files locally. [e16569d](https://github.com/uiowa/uiowa/pull/3383/commits/e16569da74711e1a65c52cb5e689e130436ec7d6)

This might not be possible: https://github.com/uiowa/uiowa/pull/3383#issue-861508526

**Comment by joewhitsitt** [comment]
@richardbporter should I start holding on site provisions?

**Comment by richardbporter** [comment]
> @richardbporter should I start holding on site provisions?

No. These files might not be necessary at all. Go ahead.

**Comment by richardbporter** [comment]
>Investigate BLT environment detector usage and determine if there is a way to set AH vars in DDev without breaking stuff. That is, it'd be nice to expand the environment variables used in the drush config files locally. [e16569d](https://github.com/uiowa/uiowa/pull/3383/commits/e16569da74711e1a65c52cb5e689e130436ec7d6)

Since we don't need to create site-specific config files anymore, I don't think its necessary to do this. For the record, BLT makes an exception for Lando so if we want to set AH env vars locally while still having EnvironmentDetector consider it local, see [7ec4f99](https://github.com/uiowa/uiowa/pull/4910/commits/7ec4f9956b39221690610cb40e3319b17dbc183a). Otherwise we have to override almost every EnvironmentDetector method.
